apps/condo/domains/ticket/constants.js

```diff
@@ -18,6 +18,12 @@ const TICKET_STATUS_TYPES = [
     CLOSED_STATUS_TYPE,
 ]
 
+/*
+    `sender` is an internal field, that don't need to be displayed in UI.
+    It's not participating in Ticket changes UI for customer.
+ */
+const OMIT_TICKET_CHANGE_TRACKABLE_FIELDS = ['sender']
```

Comment:
I think we can use sender to track whom trigger the change

Reply:
Since `TicketChange` is a schema for end-users, it seems to me, that only changes in business-related data fall into  use cases. It will make no sense to end-user, when it will see records without changes in data.
I'm remembering, that you told me about variant to omit registered changes of `Ticket` items with only changed `sender` during rendering, but what are cases, that this extra added complexity will solve?
AFAIK, `TicketChange` is not intended to be created for some internal supervision, — for this case we have `TicketHistoryRecord`.

Reply:
Added `sender` field in `ed7241be`, as a follow up our conversation.